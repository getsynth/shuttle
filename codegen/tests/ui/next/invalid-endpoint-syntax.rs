shuttle_codegen::app! {
    #[shuttle_codegen::endpoint(method = get, route = "/hello" extra = abundant)]
    async fn hello() -> &'static str {
        "Hello, World!"
    }

    #[shuttle_codegen::endpoint(method = get, route = "/goodbye", invalid)]
    async fn goodbye() -> &'static str {
        "Goodbye, World!"
    }
}
