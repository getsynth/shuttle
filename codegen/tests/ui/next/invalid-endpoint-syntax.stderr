error: expected `,`

         = help:

 --> tests/ui/next/invalid-endpoint-syntax.rs:2:64
  |
2 |     #[shuttle_codegen::endpoint(method = get, route = "/hello" extra = abundant)]
  |                                                                ^^^^^

error: expected `=`

         = help:

 --> tests/ui/next/invalid-endpoint-syntax.rs:7:74
  |
7 |     #[shuttle_codegen::endpoint(method = get, route = "/goodbye", invalid)]
  |                                                                          ^

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/ui/next/invalid-endpoint-syntax.rs:11:2
   |
11 | }
   |  ^ consider adding a `main` function to `$DIR/tests/ui/next/invalid-endpoint-syntax.rs`
